admin/javascript/src/LeftAndMain.js

```diff
@@ -227,21 +227,18 @@ $.entwine('ss', function($) {
 			var self = this,
 				basePath = getUrlPath($('base')[0].href);
 
-			// Avoid adding a double slash if the base path is '/'
+			// Remove trailing / from base
 			if (basePath[basePath.length - 1] === '/') {
-				basePath += 'admin';
-			} else {
-				basePath = '/admin';
+				basePath = basePath.substring(0, basePath.length - 1);
 			}
-
 			router.base(basePath);
 
 			// Register all top level routes.
 			Config.getTopLevelRoutes().forEach((route) => {
-				router(`/${route}/*`, (ctx, next) => {
-					// If the page isn't ready or the request hasn't come from 'loadPanel'
-					// then don't PJAX load the panel. Note: __forceReferer is set by 'loadPanel' only.
-					if (document.readyState !== 'complete' || typeof ctx.state.__forceReferer === 'undefined') {
```

Comment:
having the __forceReferer check here broke going "back" to the initial state, since it wasn't loaded via loadPanel().

Instead, sub-section loaders can conditionally avoid calling next() in their route handlers to prevent bubbling back up to this root controller.